### ISSUE TITLE
Colors Constants for Discord Embed

### ISSUE THREAD

**Issue by javis86** [ToshiroZ/Discord-Webhook#2]
I' ve just added to my package the following class:

```cs
public static class Colors
    {
        public static readonly DColor Red = new DColor(231, 76, 70); //https://www.spycolor.com/e74c3c#
        public static readonly DColor DarkRed = new DColor(153,45,34); //https://www.spycolor.com/992d22#
        public static readonly DColor Purple = new DColor(155,89,182); //https://www.spycolor.com/9b59b6#
        public static readonly DColor Green = new DColor(46,204,113); //https://www.spycolor.com/2ecc71#
        public static readonly DColor Orange = new DColor(230,126,34); //https://www.spycolor.com/e67e22#
        public static readonly DColor Grey = new DColor(179,177,175); //https://www.spycolor.com/b3b1af#
        public static readonly DColor Black = new DColor(255,255,255);
    }
```
I could add it to this library and add some other colors.

**Comment by ToshiroZ** [comment]
That would be nice. If you want to make a pull request with it I will accept it. If you want I can add it myself though

**Comment by javis86** [comment]
Done!

**Comment by javis86** [comment]
Merged in #4
